R/ReactantExtra/build_tarballs.jl

```diff
@@ -0,0 +1,222 @@
+    ls
+    cd ..
+
+    mv baz/output/bazel .local/bin/bazel
+fi
+
+#mkdir -p .julia
+#cd .julia
+
+#export JULIA_PATH=/usr/local/julia
+#export PATH=$JULIA_PATH/bin:$PATH
+
+#	wget -O julia.tar.gz "https://julialang-s3.julialang.org/bin/musl/x64/1.8/julia-1.8.5-musl-x86_64.tar.gz"
+	
+#	mkdir -p "$JULIA_PATH"; 
+#	tar -xzf julia.tar.gz -C "$JULIA_PATH" --strip-components 1; 
+#	rm julia.tar.gz; 
+
+# cd ..
+
+mkdir .bazhome
+export HOME=`pwd`/.bazhome
+
+export BAZEL_CXXOPTS="-std=c++17"
+BAZEL_FLAGS=()
+BAZEL_BUILD_FLAGS=()
+
+# don't run out of temporary space
+BAZEL_FLAGS+=(--output_user_root=/workspace/bazel_root)
+
+BAZEL_BUILD_FLAGS+=(--jobs ${nproc})
+
+BAZEL_BUILD_FLAGS+=(--verbose_failures)
+BAZEL_BUILD_FLAGS+=(--cxxopt=-std=c++17 --host_cxxopt=-std=c++17)
+BAZEL_BUILD_FLAGS+=(--cxxopt=-DTCP_USER_TIMEOUT=0)
+BAZEL_BUILD_FLAGS+=(--check_visibility=false)
+BAZEL_BUILD_FLAGS+=(--build_tag_filters=-jlrule)
+BAZEL_BUILD_FLAGS+=(--experimental_cc_shared_library)
+
+# Disable enabled-by-default TensorFlow features that we don't care about.
+BAZEL_BUILD_FLAGS+=(--define=no_aws_support=true)
+BAZEL_BUILD_FLAGS+=(--define=no_gcp_support=true)
+BAZEL_BUILD_FLAGS+=(--define=no_hdfs_support=true)
+BAZEL_BUILD_FLAGS+=(--define=no_kafka_support=true)
+BAZEL_BUILD_FLAGS+=(--define=no_ignite_support=true)
+BAZEL_BUILD_FLAGS+=(--define=grpc_no_ares=true)
+
+BAZEL_BUILD_FLAGS+=(--define=llvm_enable_zlib=false)
+BAZEL_BUILD_FLAGS+=(--verbose_failures)
+
+if [[ "${bb_full_target}" == *darwin* ]]; then
+    BAZEL_BUILD_FLAGS+=(--define=build_with_mkl=false --define=enable_mkl=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_32_1=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_64_1=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_64_2=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_aarch64=false)
+    BAZEL_BUILD_FLAGS+=(--define=clang_macos_x86_64=true)
+    BAZEL_BUILD_FLAGS+=(--macos_minimum_os=10.14)
+    export MACOSX_DEPLOYMENT_TARGET=10.14
+    BAZEL_BUILD_FLAGS+=(--action_env=MACOSX_DEPLOYMENT_TARGET=10.14)
+    BAZEL_BUILD_FLAGS+=(--host_action_env=MACOSX_DEPLOYMENT_TARGET=10.14)
+    BAZEL_BUILD_FLAGS+=(--repo_env=MACOSX_DEPLOYMENT_TARGET=10.14)
+    BAZEL_BUILD_FLAGS+=(--test_env=MACOSX_DEPLOYMENT_TARGET=10.14)
+    BAZEL_BUILD_FLAGS+=(--noincompatible_remove_legacy_whole_archive)
+    BAZEL_BUILD_FLAGS+=(-s)
+    env
+fi
+
+if [[ "${bb_full_target}" == *freebsd* ]]; then
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_32_1=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_64_1=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_64_2=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_aarch64=false)
+    BAZEL_BUILD_FLAGS+=(--define=freebsd=true)
+fi
+
+if [[ "${bb_full_target}" == *i686* ]]; then
+    BAZEL_BUILD_FLAGS+=(--define=build_with_mkl=false --define=enable_mkl=false)
+fi
+
+# julia --project=. -e "using Pkg; Pkg.instantiate(); Pkg.add(url=\"https://github.com/JuliaInterop/Clang.jl\", rev=\"vc/cxx_parse2\")"
```

Comment:
I think it's part of Clang 0.18

Reply:
Sadly we don't use that right now because I can't get julia to be usable in BB, ironically